platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -194,10 +194,7 @@ private void registerMbean() {
         }
       } catch (Exception e) {
         // objectName is not always non-null because new ObjectName(...) can throw an exception
```

Comment:
Can't ojbectName be null here?

Reply:
objectName will always be non-null if mBeanServer is also non-null. I removed the comment